pyne/r2s.py

```diff
@@ -14,9 +14,9 @@
 def irradiation_setup(flux_mesh, cell_mats, alara_params, tally_num=4,
                       geom=None, num_rays=10, grid=False, flux_tag="n_flux",
                       fluxin="alara_fluxin", reverse=False,
-                      alara_inp="alara_geom", alara_matlib="alara_matlib",
+                      alara_inp="alara_inp", alara_matlib="alara_matlib",
                       output_mesh="r2s_step1.h5m", output_material=False,
-                      sub_voxel=False):
+                      decay_times=['1 s'], sub_voxel=False):
```

Comment:
You should not put mutable containers (such as a list) as the default value of a kwarg.  Please use a tuple instead.